ios/Classes/PluginExtensions/SwiftOneginiPlugin+Auth.swift

```diff
@@ -44,61 +45,76 @@ extension SwiftOneginiPlugin: OneginiPluginAuthProtocol {
             OneginiModuleSwift.sharedInstance.authenticateUser(nil, callback: result)
         }
     }
-    
+
+    func authenticateUserImplicitly(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
+        guard let arg = call.arguments as? [String: Any]? else {
```

Comment:
I think this should be `guard let arg = call.arguments as? [String: Any] else {`

If the result from `as? <Type>?` is nil, then the guard will fail and you end up in the `else` anyway. So it's better to cast it to `[String: Any]` so that type is fully usuable below it. 

See for instance line 60: 
`let scopes = arg?["scopes"] as? [String]` could be `let scopes = arg["scopes"] as? [String]`

Reply:
sounds about right; updated the logic